### ISSUE TITLE
Tokenization of fractions

### ISSUE THREAD

**Issue by nschneid** [UniversalDependencies/UD_English-EWT#337]
For fractions like "1/2", EWT tokenizes into 3 words (with the exception of one instance that looks like an error), and this is consistent with the comment under [`NumType=Frac`](https://universaldependencies.org/en/feat/NumType.html#frac-fractional-numbers). But GUM treats the whole fraction as one token.

Note that dates with slashes are considered one token in EWT.

**Comment by amir-zeldes** [comment]
I think I like for it to be consistent with 'vulgar' fraction glyphs, like "½", and the word "half". We also don't usually pronounce this as "one divided by two", which would justify compositional treatment as 3 tokens.

**Comment by nschneid** [comment]
What does OntoNotes do? I honestly think arguments could be made either way for many kinds of numeric expressions with substructure and punctuation, including phone numbers and so on, so there should be a strong reason to depart from established tokenization policies.

**Comment by nschneid** [comment]
> We also don't usually pronounce this as "one divided by two", which would justify compositional treatment as 3 tokens.

A policy based strictly on pronunciation would say that "one half" is two words, so it should be two tokens, "1 /2". But that looks pretty silly. :) I think pronunciation is just a heuristic; it should not always be determinative for the treatment of numerals and other kinds of notation.

**Comment by nschneid** [comment]
Looking again at what I thought was an error, it is

> Otherwise a 1/2 brick will be fine:

Now I'm not sure this is an error. It is not "a one half brick" but rather "a half brick" (i.e. a partial brick), so "1/2" strikes me as a nonstandard shorthand/abbreviation for "half" in this context. Thinking of leaving it as one (adjective) token and adding `Abbr=Yes`.

**Comment by amir-zeldes** [comment]
ON is not totally consistent on this, but the vast majority of cases follow the WSJ subcorpus in having single tokens "1/2" etc. The exception is the broadcast news subcorpus (all 7 subtokenized cases come from bn, vs. 59 kept tokenized together).

**Comment by nschneid** [comment]
Hmm. Maybe Penn changed their conventions and never updated WSJ within OntoNotes. I wonder if https://catalog.ldc.upenn.edu/LDC2015T13 changed this for WSJ.

**Comment by amir-zeldes** [comment]
No, that was not updated, I just checked (and I don't think it should have been, I think "1/2" as one token is correct). 

It's also not just WSJ - it's all OntoNotes subcorpora, except for BN.

**Comment by nschneid** [comment]
Bringing the discussion about fractions back here:

_Originally posted by @dan-zeman in https://github.com/UniversalDependencies/docs/issues/885#issuecomment-1198402608_
>> @dan-zeman Would you advocate `NumType=Frac` for "3%" where the percent sign is included in the token, but something else where they are split?
>
> Yeah, I guess I would. (Also I think I'm responsible for the current usage of `NumType=Frac` for _3%_ in Spanish and Catalan – if I recall it correctly, they had a specific tag in the original annotation of AnCora, and I wanted to convert it to something different than normal numbers.)

So I think @dan-zeman is saying that, in Czech, where "3 %" acts like number+noun, it is NOT `NumType=Frac`, but rather a cardinal number plus "%", and it is through an analytical construction that we get the proportional meaning. Whereas if "3%" acts like an adjective and remains as one token, it is `NumType=Frac` because the proportional meaning is part of the morphology/spelling.

Is this a strict principle to be followed in general, that any meaning deriving from an analytical combination of words cannot determine the morphological features? Wouldn't that include, e.g., passive voice in English (`Voice=Pass`), since it is morphologically the past participle form of the verb, and we only know it is passive from the wider analytical construction?

**Comment by nschneid** [comment]
Or "2 of 3", but probably not "two thirds"
